content/francais/python-turtle/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Python : Coder avec Turtle"
+description: "Apprenez à dessiner sur un canevas avec Python et Turtle"
+date: 2024-09-17T13:24:17-07:00
```

Comment:
@oliver-zhang-neuvo  should date be modified? asking as general question. For the manual translation I didn't edit since I thought this was "workshop date created". Since I was just editing the content language, I kept it as-is

Reply:
this is not by tool but I manually edited since I saw few other PR's where authors edited the date.